modules/osd-create-cluster-red-hat-account.adoc

```diff
@@ -0,0 +1,125 @@
+.. Click *Next*.
+
+ifdef::osd-quickstart[]
+. Select *Run on Amazon Web Services* or *Run on Google Cloud Platform* to choose a cloud infrastructure provider 
+endif::osd-quickstart[]
+ifdef::osd-on-aws[]
+. Select *Run on Amazon Web Services* 
+endif::osd-on-aws[]
+ifdef::osd-on-gcp[]
+. Select *Run on Google Cloud Platform* 
+endif::osd-on-gcp[]
+and click *Next*.
+
+. On the *Cluster details* page, provide a name for your cluster and specify the cluster details:
+.. Add a *Cluster name*.
+.. Select a cluster version from the *Version* drop-down menu.
+.. Select a cloud provider region from the *Region* drop-down menu.
+.. Select a *Single zone* or *Multi-zone* configuration.
+.. Select a *Persistent storage* capacity and state the number of *Load balancers* that you require for your cluster.
+.. Leave *Enable user workload monitoring* selected to monitor your own projects in isolation from Red Hat Site Reliability Engineer (SRE) platform metrics. The option is enabled by default.
+.. Optional: Select *Enable additional etcd encryption* if you require etcd key value encryption. With this option, the etcd key values are encrypted, but not the keys. The option is in addition to the control plane storage encryption that encrypts the etcd volumes in {product-title} clusters by default.
++
+[NOTE]
+====
+By enabling etcd encryption for the key values in etcd, you will incur a performance overhead of approximately 20%. The overhead is a result of introducing this second layer of encryption, in addition to the default control plane storage encryption that encrypts the etcd volumes. Consider enabling etcd encryption only if you specifically require it for your use case.
+====
+.. Click *Next*.
+
+. On the *Default machine pool* page, select a *Compute node instance type* and a *Compute node count*. The number and types of nodes that are available depend on your {product-title} subscription. If you are using multiple availability zones, the compute node count is per zone.
++
+[NOTE]
+====
+After your cluster is created, you can change the number of compute nodes in your cluster, but you cannot change the compute node instance type. The number and types of nodes available to you depend on your {product-title} subscription.
+====
+
+. Optional: Expand *Edit node labels* to add labels to your nodes. Click *Add label* to add more node labels.
+
+. In the *Cluster privacy* dialog, select *Public* or *Private* to use either public or private API endpoints and application routes for your cluster.
+
+. In the *CIDR ranges* dialog, configure custom classless inter-domain routing (CIDR) ranges or use the defaults that are provided.
++
+[IMPORTANT]
+====
+CIDR configurations cannot be changed later. Confirm your selections with your network administrator before proceeding.
+
+If the cluster privacy is set to *Private*, you will not be able to access your cluster until you configure private connections in your cloud provider.
```

Comment:
```suggestion
If the cluster privacy is set to *Private*, you are not able to access your cluster until you configure private connections in your cloud provider.
```

Reply:
I have opted for "you cannot access your cluster" as a present tense replacement here. Thanks for highlighting this.